web/src/search_suggestion.ts

```diff
@@ -752,42 +754,74 @@ function get_operator_suggestions(last: NarrowTerm): Suggestion[] {
     });
 }
 
+// One search suggestion can include multiple search terms, based on what's
+// in the search bar.
+type SuggestionLine = Suggestion[];
+function suggestion_search_string(suggestion_line: SuggestionLine): string {
+    const search_strings = [];
+    for (const suggestion of suggestion_line) {
+        if (suggestion.search_string !== "") {
```

Comment:
One way we can get empty search strings here is when creating an `Attacher` with a `base` that's an empty term. I decided it was cleaner to filter it out here than try to remove it somewhere else. Does that seem reasonable?